pyanaconda/kickstart.py

```diff
@@ -2121,6 +2123,16 @@ def runPreScripts(scripts):
 
     log.info("All kickstart %%pre script(s) have been run")
 
+def runPreInstallScripts(scripts):
+    preInstallScripts = [s for s in scripts if s.type == KS_SCRIPT_PREINSTALL]
+
+    if len(preInstallScripts) == 0:
```

Comment:
``if not any(preInstallScripts):`` would do the same, so in case you (too) like it better, you can use it here.